### PR TITLE
Update validation project for CMSIS-Toolbox 2.5.0

### PR DESCRIPTION
**Pull request by VladimirUmek**

### CODE REVIEW

File: Project/vcpkg-configuration.json [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
     "arm:tools/ninja-build/ninja": "^1.10.2",
-    "arm:compilers/arm/armclang":"^6.20.0",
+    "arm:compilers/arm/armclang": "^6.20.0",
     "arm:compilers/arm/arm-none-eabi-gcc": "^13.2.1",
     "arm:compilers/arm/llvm-embedded": "^17.0.1-0",
-    "arm:tools/open-cmsis-pack/cmsis-toolbox": "^2.2.0-0",
-    "arm:models/arm/avh-fvp": "^11.22.39"
+    "arm:tools/open-cmsis-pack/cmsis-toolbox": "^2.5.0-0",
+    "arm:models/arm/avh-fvp": "^11.22.39",
+    "arm:debuggers/arm/armdbg": "^6.2.0"
```
Comment (jkrech) [inline Project/vcpkg-configuration.json:17]: are you actually using the debugger?

**Review by jkrech — APPROVED** [review]
Looks good to me. Not sure you need the armdebugger in vcpkg-configuration.json.

File: Project/vcpkg-configuration.json (hunk 1 above)
Comment (VladimirUmek) [reply Project/vcpkg-configuration.json:17]: Well, one can use it, yes.
This commit does not provide launch configuration for VS Code with Arm Keil Studio Pack extensions, but I will re-add it (the old one was deleted). With correct launch configuration FVP debugging will work out of the box.

Comment (VladimirUmek) [reply Project/vcpkg-configuration.json:17]: The launch configuration is now part of this PR

**Pull request merged**